purview/utils/general.py

```diff
@@ -92,6 +92,9 @@ def get_neo4j_node(resource_name, uid):
     # To prevent a ciruclar import, we must import this here
     from purview.models import all_models
 
+    if resource_name.lower() == 'kojicontainerbuild':
```

Comment:
Could you please delete this if statement and add `KojiContainerBuild` to `all_models`. That should remove the need for this.